index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pl'>
+
+				<p>Jesteśmy społecznością osób, którym nie jest obojętny rozwój i stan światowej sieci internetowej oraz wyrażamy <strong>głęboki niepokój</strong> w związku z technologią <a href='https://ampproject.org/'>Accelerated Mobile Pages</a> („AMP” – Przyśpieszone Mobilne Strony Internetowe). AMP to projekt, który rzekomo ma na celu poprawę wygody użytkowników podczas korzystania z sieci internetowej.</p>
+				
+				<p>Tymczasem AMP zatrzymuje użytkowników wewnątrz domeny Google oraz przechwytuje ruch z innych stron internetowych na rzecz Google. W skali miliardów użytkowników jest to działanie, które <strong>jeszcze bardziej umocni dominującą pozycję Google w sieci internetowej</strong>.</p>
+				
+				<p>Zdajemy sobie sprawę z problemu powolnego ładowania się witryn internetowych, zwłaszcza w porównaniu do własnościowych rozwiązań takich jak <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> i <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Wydawcy treści (zwłaszcza w sektorze mediów informacyjnych) od dawna zmagają się z trudnymi wyborami i niesłusznymi pobudkami, co prowadzi do złych decyzji i kompromisów, a te z kolei skutkują pogorszeniem wygody użytkowników.</p>
+				
+				<p>Wyszukiwarki to potężne narzędzia, które mają na tyle duże wpływy, żeby móc rozwiązać ten problem. Jednakże Google zdecydowało się stworzyć wyróżniającą się pozycję na samej górze wyników wyszukiwania (dla artykułów) oraz dodawać specjalną ikonę <i>błyskawicy</i> (dla pozostałych treści), ale tylko dla wydawców korzystających z technologii kontrolowanej przez Google, udostępnianej przez Google w ramach ich infrastruktury, w ramach adresu URL Google oraz wewnątrz interfejsu stworzonego przez Google (<a href='https://developers.google.com/search/docs/guides/about-amp'>źródło</a>).</p>
+				
+				<p>Idea AMP nie jest sama w sobie problematyczna, ale dwa aspekty jej implementacji umacniają Google na pozycji <i>de facto</i> standardowej platformy udostępniającej treść w sieci internetowej, zwłaszcza że Google próbuje zwiększyć użycie AMP:</p>
+				
+				<ol>
+					<li>Treść, która dostępna jest wewnątrz AMP (a tym samym wewnątrz domeny Google), otrzymuje dodatkową promocję w ramach wyników wyszukiwania, w tym: najwyższą pozycję dla artykułów informacyjnych.</li>
```

Comment:
> [...] w tym: najwyższą pozycję dla artykułów informacyjnych.

There should be no colon if one is not listing anything, and it's only one example.

Secondly, You may consider using semicolon in that listing (I mean `ol > li` listing) instead dot, at the end of list item. See [Wikipedia/Średnik](https://pl.wikipedia.org/wiki/%C5%9Arednik).

Reply:
You gave me a hard time thinking about this one, but I changed dots to semicolons.